public/js/sessionList.js

```diff
@@ -0,0 +1,79 @@
+const openEditSession = (id) => {
+  // eslint-disable-next-line no-underscore-dangle
+  window.location.href = `${window.location.origin}/views/sessionForm.html?_id=${id}`;
+};
+const openNewSession = () => {
+  window.location.href = `${window.location.origin}/views/sessionForm.html`;
+};
+
+window.onload = () => {
+  const navButton = document.getElementById('sessionsNav');
+  navButton.classList.add('activePage');
+
+  const tableContent = document.getElementById('session-table');
+
+  const addSession = document.getElementById('addSession');
+  addSession.onclick = openNewSession;
+
+  const deleteSession = (item, event) => {
+    event.stopPropagation();
+    // eslint-disable-next-line no-undef
+    updateModalInfo(
+      'You are about to delete a Session between:',
+      `${item.postulant.firstName} ${item.postulant.lastName}
+    and ${item.psychologist.firstName} ${item.psychologist.lastName}`,
+    );
+    // eslint-disable-next-line no-undef
+    openModal();
+    document.getElementById('cancel-button').onclick = () => {
+    // eslint-disable-next-line no-undef
+      closeModal();
+    };
+    document.getElementById('procced-button').onclick = () => {
+      // eslint-disable-next-line no-underscore-dangle
+      const url = `${window.location.origin}/api/sessions/${item._id}`;
+      fetch(url, {
+        method: 'DELETE',
+      })
+        .then((response) => {
+          response.json();
```

Comment:
you should check if the deletion was ok before redirecting the page. Anyway, you should close the modal and get the new session list instead of redirecting the page.

Reply:
fixed!